test/snippet/alphabet/cigar/cigar_op.cpp

```diff
@@ -0,0 +1,27 @@
+#include <seqan3/alphabet/cigar/cigar_op.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+//! [general]
+// Initialze an seqan3::cigar_op:
+cigar_op match{'M'_cigar_op};
+
+// Usually you want to use the cigar_op directly in a cigar element
+// cigar my_cigar_elem1{match, 20u};
+
+// you print cigar_op and cigar values:
+std::cout << match.to_char() << std::endl;          // M
```

Comment:
Please move this before the out-commented code before and close the `//! [general]` block so it does not display the out-commented code. Then add a todo that we move the general part once we can comment those lines in.